spec/models/piece_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'

```

Comment:
The tests all pass 👍 

I would also suggest testing the cases where false is also true, which should be a simple add